example/credentials_ssh.tf

```
@@ -0,0 +1,12 @@
+resource "jenkins_credential_ssh" "global" {
+  name     = "some-id"
+  username = "example-username"
+  privatekey = file(/some/path/id_rsa)
+}
+
+resource "jenkins_credential_ssh" "folder" {
+  name       = "some-id"
+  folder     = jenkins_folder.example.id
+  username   = "example-username"
+  privatekey = file(/some/path/id_rsa)
+}
```

Obviously this is a placeholder, but still probably good to have it be valid syntax :)

```
Error: Invalid expression
│ 
│   on credentials_ssh.tf line 4, in resource "jenkins_credential_ssh" "global":
│    4:   privatekey = file(/some/path/id_rsa)
│ 
│ Expected the start of an expression, but found an invalid expression token.
```

```suggestion
resource "jenkins_credential_ssh" "global" {
  name     = "some-id"
  username = "example-username"
  privatekey = file("/some/path/id_rsa")
}

resource "jenkins_credential_ssh" "folder" {
  name       = "some-id"
  folder     = jenkins_folder.example.id
  username   = "example-username"
  privatekey = file("/some/path/id_rsa")
}
```